src/main/java/frc/robot/commands/AidenIntakeCommand.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.RollerSubsystem;
+
+/**
+ * Intakes 1 gamepiece with Roller Mech. This command is to be used during autonamous.
+ */
+
+public class AidenIntakeCommand extends CommandBase {
+    private final RollerSubsystem rollerSubsystem;
+    private boolean limitswitchUp;
+    private double power;
+
+    public AidenIntakeCommand(RollerSubsystem rollerSubsystem){
+        power = 0.0;
+        this.rollerSubsystem = rollerSubsystem;
+        limitswitchUp = rollerSubsystem.isLimit();
+        addRequirements(rollerSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        System.out.println("Roller to intake");
+    } 
+    
+    @Override
+    public void execute() {
+        //let rollers turn intil a game piece is in(has pressed onto limit switch)
+        if (limitswitchUp){
```

Comment:
Is it intentional that you only query this once? You could instead do
```java
if (rollerSubsystem.isLimit()) {
```
and remove the field.